deepchem/feat/molecule_featurizers/grover_featurizer.py

```diff
@@ -143,9 +152,26 @@ def _make_mol_graph(self, mol: RDKitMol) -> GraphData:
 
                 edge_index.extend([[a1, a2], [a2, a1]])
 
+                b1 = n_bonds
```

Comment:
Can you add some comments here explaining what these lines do? It's a bit confusing to read

Reply:
added